src/Appwrite/Utopia/Database/Validator/Queries/Messages.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Appwrite\Utopia\Database\Validator\Queries;
+
+class Messages extends Base
+{
+    public const ALLOWED_ATTRIBUTES = [
+        'to',
+        'providerId',
+        'deliveredAt',
+        'deliveredTo',
+        'deliveryErrors',
+        'status',
+        'description',
+        'data',
+        'search'
```

Comment:
we typically don't allow querying on search. Instead, it's a separate param on the endpoint.